### ISSUE TITLE
osx edge ServiceBuilder bug in constructor() on javascript node selenium-webdriver 4.0.0-alpha.7

### ISSUE THREAD

**Issue by CcSimple** [SeleniumHQ/selenium#8574]
## 🐛 Bug Report

const EDGE_CHROMIUM_BROWSER_NAME = "msedge";
function locateSynchronously(browserName) {
  browserName = browserName || Browser.EDGE;
 # 🐛 Browser.EDGE is "MicrosoftEdge"  not  "msedge"
  if (browserName === EDGE_CHROMIUM_BROWSER_NAME) {
    return io.findInPath(EDGEDRIVER_CHROMIUM_EXE, true);
  }
  return process.platform === 'win32'
      ? io.findInPath(EDGEDRIVER_LEGACY_EXE, true) : null;
}

## Environment

OS:  OSX
Browser:  EDGE
Browser version:  84.0.522.44
Browser Driver version:  msedgedriver_mac64_84.0.522.44
Language Bindings version:  
Selenium Grid version (if applicable):

**Comment by harsha509** [comment]
Hi @CcSimple ,

Thanks for raising the issue. 
This is not a bug, the code is expected to support 'Microsoft Chromium Edge' and 'Microsoft Edge legacy'(in windows machine)

From the above data, I can see you're trying to run the tests in Edge Chromium. For the test to consider Microsoft Chromium edge driver I.e `msedge` in your machine, you need to provide edgeOption `opts.setEdgeChromium(true);`

Hope this helps.

Regards,
Harsha

**Comment by corevo** [comment]
I can't understand from the issue what the problem is, all I can see is pasted code from our code base.

I'll need you to kindly explain in the body of the issue what is the bug (in plain English), and what is the expected behavior, I can see that @Harsha509 kindly tried to understand from the fragments what you meant to say, but I can't fix a bug that I don't understand.

What I can tell you is that in order to not break the API, we've decided to keep the current constants refer to old Edge and only add new constants to refer to chromium Edge, it is by design, and once we will ship the next major version we might consider breaking this paradigm.

**Comment by diemol** [comment]
I will close this since the OP didn't reply the comments made above. 
Feel free to reopen or open a new issue with a more detailed explanation and scripts to reproduce the issue.
